deploy/crds/hco00.crd.yaml

```diff
@@ -65,6 +65,8 @@ spec:
                 parallelOutboundMigrationsPerNode: 2
                 progressTimeout: 150
               uninstallStrategy: BlockUninstallIfWorkloadsExist
+              virtualMachineOptions:
+                disableFreePageReporting: null
```

Comment:
this smells like a bug to me:
I'd expect `disableFreePageReporting: true` as the default also at this level.

Reply:
and indeed in https://github.com/kubevirt/hyperconverged-cluster-operator/actions/runs/5462811618/jobs/9942676734?pr=2356
I see:
```
git difftool -y --trust-exit-code --extcmd=./hack/diff-csv.sh
--- /tmp/git-blob-8QLJGp/hco.kubevirt.io_hyperconvergeds.yaml	2023-07-05 09:49:33.189257961 +0000
+++ config/crd/bases/hco.kubevirt.io_hyperconvergeds.yaml	2023-07-05 09:48:53.035161657 +0000
@@ -68,3 +68,3 @@
               virtualMachineOptions:
-                disableFreePageReporting: null
+                disableFreePageReporting: true
             description: HyperConvergedSpec defines the desired state of HyperConverged
fatal: external diff died, stopping at config/crd/bases/hco.kubevirt.io_hyperconvergeds.yaml
```
so this is probably just the result of a wrong rebase.

Reply:
It should definitely be better now. Thanks :+1: